README.md

```diff
@@ -10,12 +10,23 @@ __Be sure to understand the security implications of backing up the secrets from
 
 # Instructions
 
+## ADB Backup
+
 Make an adb backup of FreeOTP, (details on how to do this are currenly left as an excercise for the reader. While you are doing the backup, consider the above warning!). Open the adb backup file (.ab) with this app. Read the displayed qrcodes with the authenticator app on your other phone.
 
 The author has tested the app with current Google Chrome and Mozilla Firefox on Ubuntu 16.04 in April 2017. Other browsers probably do not work. Generated qrcodes have been succesfully imported in FreeOTP and Google Authenticator on Android and Microsoft Authenticator on Windows Phone 8, Windows Phone 10 and Android.
 
 This app can be opened directly at https://rawgit.com/viljoviitanen/freeotp-export/master/export.html 
 
+## XML Backup
+
+If your phone is already rooted you can directly backup the XML file containing all your accounts.
+Just use a root explorer (something like [Amaze File Manager](https://play.google.com/store/apps/details?id=com.amaze.filemanager)) to copy the tokens.xml file from `/data/data/org.fedorahosted.freeotp/shared_prefs/tokens.xml` to your phones internal storage.
```

Comment:
Please remove the link to Amaze File Manager, it's enough to say that the file can be extracted with a (root) file explorer.